tests/appsec/test_shell_execution.py

```diff
@@ -3,92 +3,90 @@
 # Copyright 2021 Datadog, Inc.
 
 import json
-from utils import coverage, interfaces, weblog, features
+from utils import coverage, interfaces, weblog, features, irrelevant
+from utils._context.core import context
```

Comment:
We usually import `context` from `utils` rather than `utils._context.core`.

Reply:
Can you change it in your PR instead? I'm afraid changing the commits will invalidate the approvals :p

Reply:
(great power, great responsabilties 😄 🕷️ )